apps/demo/tailwind.config.js

```diff
@@ -0,0 +1,14 @@
+const { createGlobPatternsForDependencies } = require('@nrwl/angular/tailwind');
+const { join } = require('path');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    join(__dirname, 'src/**/!(*.stories|*.spec).{ts,html}'),
```

Comment:
I guess this is generated by nrwl? But do we need to exclude `.stories` files we don't have?

Reply:
Fixed by removing storybook config